src/internal/modules/charge-information/lib/helpers.js

```diff
@@ -85,8 +85,8 @@ const getLicencePageUrl = async licence => {
 
 const isCurrentAddress = invoiceAccountAddress => invoiceAccountAddress.dateRange.endDate === null;
 
-const getCurrentBillingAccountAddress = billingAccount => billingAccount.invoiceAccountAddresses
-  .find(isCurrentAddress);
+const getCurrentBillingAccountAddress = billingAccount => billingAccount && billingAccount.invoiceAccountAddresses ? billingAccount.invoiceAccountAddresses
+  .find(isCurrentAddress) : undefined;
 
```

Comment:
This could maybe be simplified a bit with lodash `get`
```
const getCurrentBillingAccountAddress = billingAccount => get(billingAccount, 'invoiceAccountAddresses', []).find(isCurrentAddress);
```
I think as a bonus it will increase the test coverage by removing the ternary operator.